docs/firmware/fw.md

```diff
@@ -58,3 +58,18 @@ following documents.
 
 [FW_JUMP]: fw_jump.md
 [FW_PAYLOAD]: fw_payload.md
+
+Providing different payloads to OpenSBI Firmware
+------------------------------------------------
+OpenSBI firmware can accept various payloads as an compile time options.
+Typically, these payload refer to the next stage boot loader (e.g. U-Boot)
+or supervisor operating system images (e.g. Linux). By default, OpenSBI
```

Comment:
Remove "supervisor" here. Not needed. And replace "operating system" with "kernel" or "operating system kernel".
(an OS is not a kernel and vice-versa)